scrapers.js

```diff
@@ -1686,6 +1686,32 @@ let scrapers = [
       };
     }
   },
+  {
+    country: 'ESP',
+    url: 'https://opendata.arcgis.com/datasets/48fac2d7de0f43f9af938852e3748845_0.csv',
```

Comment:
Make this the authoritative data source for ESP:

```suggestion
    url: 'https://opendata.arcgis.com/datasets/48fac2d7de0f43f9af938852e3748845_0.csv',
    priority: 1,
```